lib/bolt/config/options.rb

```diff
@@ -33,6 +33,18 @@ module Options
             "_plugin" => {
               description: "The name of the plugin.",
               type: "string"
+            },
+            "_cache" => {
+              description: "This feature is experimental. Enable plugin caching and set a time-to-live.",
+              type: "object",
+              required: ["ttl"],
+              properties: {
+                "ttl" => {
+                  description: "Time in seconds to keep the plugin cache.",
+                  type: "integer",
+                  minimum: 0
```

Comment:
It seems like setting a minimum of 0 would effectively make the cache useless?

Reply:
It's a way to turn off a default cache. Say I want to cache all my plugins except one, setting it to 0 lets users disable cache for one or a few plugins.

Reply:
Ah, that's helpful. I would add that to the docs so users can discover that behavior more easily.

Reply:
Will do! I also added a check for 0 so we don't waste time reading/writing cache if it's 0.

Reply:
It looks like docs are missing info about setting `ttl` to 0 to disable caching when `plugin-cache` is configured.